### PR TITLE
Fix bug 1520050 - Remove custom managers from the User model because they gene…

### PR DESCRIPTION
**Pull request by jotes**
Hey,
It looks like after some small refactorings, I was able to fix bug 1342613.

Can you look if this is mergeable @mathjazz?
Thanks in advance

### CODE REVIEW

File: pontoon/base/models.py [hunk 1]
```diff
@@ -90,152 +90,6 @@ def combine_entity_filters(entities, filter_choices, filters, *args):
...
-                'count': day['count'],
-                'project': example.entity.resource.project,
-                'translation': example,
-            })
-
-        return timeline
-
-
-class UserQuerySet(models.QuerySet):
-    def serialize(self):
```
[the remark this answers is not in the document]
Comment (jotes) [reply pontoon/base/models.py]: I was surprised too.

**Comment by jotes** [comment]
@mathjazz a good catch! Something is fishy because Travis is green (and tests for contributors shouldn't even pass).

**Comment by jotes** [comment]
@mathjazz Okay, once again you proved that we don't have to rely on Travis because we have you ;-)
Tests related to 'pontoon.contributors' app weren't executed in pytest. I fixed it.

File: pontoon/base/utils.py [hunk 2]
```diff
@@ -737,3 +739,5 @@ def readonly_exists(projects, locale):
         locale=locale,
         readonly=True,
     ).exists()
+
+
```
Comment (mathjazz) [inline pontoon/base/utils.py]: Nit: we don't need so many new lines at the end of the file.

File: pontoon/contributors/utils.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        contributor.translations_approved_count = user['approved']
+        contributor.translations_unapproved_count = user['unreviewed']
+        contributor.translations_needs_work_count = user['fuzzy']
+        contributor.user_role = contributor.role(managers, translators)
+
+    contributors_list = sorted(contributors, key=lambda x: -x.translations_count)
+    if limit:
+        contributors_list = contributors_list[:limit]
+
+    return contributors_list
```
Comment (mathjazz) [inline pontoon/contributors/utils.py]: Nit: No newline.

**Pull request merged**